### DISCUSSION TITLE
Run pixi in root mode

### DISCUSSION THREAD

**Discussion by EchoPouet** [prefix-dev/pixi#1475] [Q&A] [answered]
Hello,
I would like run my python script in root mode and I have this error:
```bash
$ sudo pixi shell
  × failed to create file `/tmp/uv-64f14bf25fb98ee4.lock`
  ╰─▶ Permission denied (os error 13)
```

I installed pixi in **/usr/local/bin/pixi** and all work in no-root mode. My OS is Ubuntu Server 22.04.

Have you got any ideas?

**Comment by EchoPouet** [comment, marked as answer]
Hoooooooooo. I found the problem, previously I ran shell in no root mode and Pixi created file /tmp/uv-64f14bf25fb98ee4.lock. In root mode, Pixi want to create or edit the same file but the the sticky permission it can't do that.

**Comment by ruben-arts** [comment]
For future users finding this issue. Pixi has no limitations with sudo, it's just not installed there by default. And as @EchoPouet explains it might try to touch the same files which might give it issues with permissions. So it's not a recommended workflow to mix `sudo` and non-`sudo` workflows. 

To get `pixi` installed in root you can simply move the binary as it is statically linked.

```
sudo cp ~/.pixi/bin/pixi /usr/bin/pixi
```

Now you should be able to run pixi as sudo
```
sudo pixi init test-sudo
cd test-sudo
sudo pixi add python
sudo pixi run python
```
